auto-update-dataset/python/repoArchivedCheck.py

```diff
@@ -5,23 +5,27 @@
 
 # it takes about 2 minutes to check 200 projects
 def main():
-    # get unique urls from pr-data.csv, we can also use cmd: `git pull -r ; cut -f1 -d, pr-data.csv | uniq`
-    pr_data_url = r"https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/pr-data.csv"
+    # get unique urls from pr-data.csv, we can also use
+    # cmd: `git pull -r ; cut -f1 -d, pr-data.csv | uniq`
+    pr_data_url = r"https://raw.githubusercontent.com/" \
+              r"TestingResearchIllinois/idoft/main/pr-data.csv"
     urls = pd.read_csv(pr_data_url, usecols=["Project URL"])
     urls = list(set([i[0] for i in urls.values.tolist()]))
     print("load data from", pr_data_url)
-    
```

Comment:
Please no unnecessary blank-space changes. If some are needed, make a separate PR with only those.

Reply:
This was a warning by Flake8, do we want to ignore or acknowledge it? 

_blank line contains whitespace_ Flake8(W293)

Reply:
Fine to address that, but can we have separate commits for these cases:
1. blank lines and whitespace
2. other non-long line issues
3. long-line issues

Can `flake8` be configured to run only one check or some checks? Dually, can it be configured to not run some checks?

Reply:
yes we can configure the `flake8` to run only specific checks or a subset of checks.
We can use --select and --ignore options in our yml workflow file.
ex. : `flake8 --ignore E501` - to ignore line too long check

Reply:
1. PR for blank lines and whitespace : #781 
2. PR for other non-long line issues : #782 
3. I will prepare a separate PR for long lines, after [discussing details](https://github.com/TestingResearchIllinois/idoft/pull/779/files#r1336459947)

Reply:
I accepted the two PRs. Let's update this one with the final changes.

Reply:
This PR is updated with the latest changes, I'm awaiting [your decision regarding the styling of parentheses](https://github.com/TestingResearchIllinois/idoft/pull/779#discussion_r1336459947).